TabLayout/tests/TabGroup-specs.js

```diff
@@ -49,16 +55,22 @@ describe('TabGroup specs', () => {
 				]}
 			/>
 		);
-
-		const expected = 3;
-		const actual = subject.find('Icon').length;
-
-		expect(actual).toEqual(expected);
+		const actualHomeIcon = screen.getByRole('group').children.item(0).children.item(1).children.item(0).textContent.codePointAt();
```

Comment:
I'm not sure this is the right approach to test it. It's highly dependent on our DOM structure which I want to avoid.
🤔 Can this case be replaced with SS test? WDYT?

Reply:
Managed to change the test so it does not depend on DOM structure.